### PR TITLE
semi-configurable selection of elements to generate tabs in admin

### PR DESCRIPTION
**Pull request by hovi**
#606 
Draft PR suggesting direction of configurable selectors.

### CODE REVIEW

File: modeltranslation/static/modeltranslation/js/tabbed_translation_fields.js [hunk 1]
```diff
@@ -212,9 +240,9 @@ var google, django, gettext;
           container.find("script").remove();
           panel = $('<div id="' + tabId + '"></div>').append(container);
           tab = $(
-            "<li" +
-              (label.hasClass("required") ? ' class="required"' : "") +
-              '><a href="#' +
+            `<li class='${selectors["tabLiClass"]} ` +
+              (label.hasClass("required") ? "required" : "") + "'" +
+              `><a class="${selectors["tabAClass"]}" href="#` +
```
Comment (last-partizan) [inline modeltranslation/static/modeltranslation/js/tabbed_translation_fields.js:245]: I think it looks good, except that now we have string templates mixed with concatenation.

But that's for another refactoring.

**Review by last-partizan — COMMENTED** [review]
I think i can merge it in this state.

Does it work fine for you with both, django and jazzmin templates?
Do you have any other improvements in mind?

**Comment by hovi** [comment]
I wouldn't merge it as it is. It's just a random draft.
It works for me, but it's not very flexible for general purposes.

In jazzmin, main select is visible, but doesn't change languages for tabs, I have to fix that..

I will think about it a bit more. I guess we could merge and then go for better separation/configuration incrementally in different PRs.

**Comment by hovi** [comment]
I think this is good enough for incremental merge. Go ahead and merge if you are okay with it.

A few notes:

- js tab library also puts it's own classes and there's no way to configure "active" tab class.
- There's problem with main select in jazzmin due to using `select2` widget. Not sure exactly why, could also be because of multiple jquery instances.
- Overall I'd say correct solution would be big refactoring with clear separation of functionality and templating. Ideal would be to make js library plugable so that js tab can be used or something else too (ex. bootstrap javascript) or make own implementation, since it should be something very very simple - just hiding/showing content based on active tab.

**Comment by last-partizan** [comment]
Merged it, thanks.

I think to refactor this is a good idea, nobody has touched it for a long time.

**Pull request merged**